Security/day03/Easy/chall-02-Ylano/front/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM php:7.4-apache
+
+# Set flag secret key, the flag is also in the config.php
+ENV FLAG=PoC{f1LTEREvErytHIngIsNIce}
```

Comment:
FLAG should not be in this repository

Reply:
delete all challenges and just give a readme to give the the url of pathwar